package.json

```diff
@@ -20,6 +20,7 @@
     "clean": "scripts/clean.sh",
     "clean:relay": "rm -rf '$TMPDIR'/RelayFindGraphQLTags-*",
     "compile": "babel src/v2 --out-dir dist/v2 -s --source-map --extensions '.js,.jsx,.ts,.tsx' --ignore src/v2/**/__tests__",
+    "compile-formatjs": "formatjs compile",
```

Comment:
Following other idioms in package.json `compile:formatjs`

Reply:
or better: `formatjs:compile`

Reply:
Much better ❤️ 
I made these changes and squashed them into the existing commits (and I updated the commit messages to reflect the instructions using these commands).